<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1284</width>
    <height>882</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="logon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>logon</string>
    </property>
   </widget>
   <widget class="QPushButton" name="service">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>service</string>
    </property>
   </widget>
   <widget class="QPushButton" name="driver">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>driver</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sheduletask">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>tasks</string>
    </property>
   </widget>
   <widget class="QPushButton" name="knowndll">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>dll</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>1211</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTableWidget" name="tableWidget_logon">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1211</width>
        <height>511</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTableWidget" name="tableWidget_service">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1201</width>
        <height>511</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QTableWidget" name="tableWidget_driver">
      <property name="geometry">
       <rect>
        <x>-50</x>
        <y>10</y>
        <width>1251</width>
        <height>501</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QTableWidget" name="tableWidget_task">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1201</width>
        <height>511</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QTableWidget" name="tableWidget_dll">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1201</width>
        <height>511</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>7</number>
      </property>
      <column>
       <property name="text">
        <string>新建列</string>
       </property>
      </column>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
